### PR TITLE
Allow Sylius 1.9

### PR DESCRIPTION
**Pull request by maximehuran**
Use MYSQL on our plugin to avoid migrations diff due to Maria DB : https://github.com/Sylius/Sylius/pull/12413
Allow Sylius ^1.8

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
         "php": "~7.4",
         "ext-mbstring": "*",
         "ext-json": "*",
-        "sylius/sylius": "~1.8.0"
+        "sylius/sylius": "^1.8"
```
Comment (jacquesbh) [inline composer.json:11]:
```suggestion
        "sylius/sylius": "~1.8.0 || ~1.9.0"
```

**Review by jacquesbh — CHANGES_REQUESTED** [review]
I would prefer this strict constraint instead. It allows patches but we don't want 1.10 yet.

**Review by jacquesbh — APPROVED** [review]
SGTM

**Review by lanfisis — APPROVED** [review]

**Comment by lanfisis** [comment]
Thanks a lot 🌯

**Pull request merged**